frontend/sw360-portlet/src/main/webapp/html/components/view.jsp

```diff
@@ -291,6 +291,11 @@
                 success: function (data) {
                     if (data.result == 'SUCCESS') {
                         componentsTable.row('#' + id).remove().draw(false);
+                        var oldValue = $('#componentCounter').text();
+                        oldValue = oldValue.substring(1, oldValue.length - 1);
+                        oldValue --;
```

Comment:
Maybe at this place it would be nicer to introduce a new variable `newValue` (just for clarity)? Otherwise it is strange to read that the `oldValue` is set...